logsource.c unused variable warnings with 3.8.1 on OS X (cur_ack_count/last_ack_count)

Release: 3.8.1
OS: OS X 10.11.6, Apple LLVM version 7.3.0 (clang-703.0.31)

When I built 3.8.1, I saw these unused variable warnings with logsource.c:

CC       lib/lib_libsyslog_ng_la-logsource.lo
lib/logsource.c:61:26: warning: unused variable 'last_ack_count' [-Wunused-variable]
  guint32 cur_ack_count, last_ack_count;
                         ^
lib/logsource.c:61:11: warning: unused variable 'cur_ack_count' [-Wunused-variable]
  guint32 cur_ack_count, last_ack_count;
          ^
2 warnings generated.

--- a/lib/logsource.c
+++ b/lib/logsource.c
@@ -70,12 +70,12 @@ _flow_control_window_size_adjust(LogSource *self, guint32 window_size_increment)
 static void
 _flow_control_rate_adjust(LogSource *self)
 {
+#ifdef SYSLOG_NG_HAVE_CLOCK_GETTIME
   guint32 cur_ack_count, last_ack_count;
   /* NOTE: this is racy. msg_ack may be executing in different writer
    * threads. I don't want to lock, all we need is an approximate value of
    * the ACK rate of the last couple of seconds.  */
 
-#ifdef SYSLOG_NG_HAVE_CLOCK_GETTIME
   if (accurate_nanosleep && self->threaded)
     {
       cur_ack_count = ++self->ack_count;

